src/aws-compile-servicecatalog.js

```diff
@@ -29,6 +29,19 @@ class AwsCompileServiceCatalog {
       || path.join(servicePath || '.', '.serverless');
     this.provider = this.serverless.getProvider('aws');
 
+    // Add custom schema properties to the AWS provider. For reference use https://github.com/ajv-validator/ajv
+    serverless.configSchemaHandler.defineProvider('aws', {
```

Comment:
is this compatibile with `serverless@1` or does this need to be a new major version? Feels like a breaking change that requires serverless@2 here.  Was there any additional documentation that needs to be updated to support serverless@2?

Reply:
This is a breaking change yes for v2

Reply:
👍 Just need to make sure we release this as a new major for this package then.

Reply:
Can we just wrap this in an `if` that checks for `serverless.configSchemaHandler`? That way, we can avoid making this a major bump.

Reply:
schema validation was added in [1.78](https://github.com/serverless/serverless/blob/master/CHANGELOG.md#1780-2020-08-03)
I don't see in the changelog at least when the provider folder location changed, but this should absolutely be a major version revision

Reply:
@decompil3d the pathway for the provider also changed which affected the unit tests, so its a breaking change there as well. I haven't found when the folder structure changed that the import path for the tests needed to change

Reply:
I think it should probably be a major bump since it is a major bump of serverless

Reply:
I've added the conditional, but serverless itself is a major bump v1 => v2 here which also to me indicates we should as well

Reply:
Ok, go ahead then :)